.github/workflows/examples-components.yml

```diff
@@ -7,17 +7,37 @@ on:
       - main
     tags:
       - component-blobby-v[0-9]+.[0-9]+.[0-9]+*
+      - component-blobby-v[0-9]+.[0-9]+.[0-9]+-*
```

Comment:
This regex isn't right. It shouldn't hold up this PR but it should be without the final *:
- `component-blobby-v[0-9]+.[0-9]+.[0-9]+`
- `component-blobby-v[0-9]+.[0-9]+.[0-9]+-.*`

Right?

Reply:
Oh so it turns out that GHA actions aren't *QUITE* regex -- `*` is more like glob when at the end. Isn't that wonderful? See `wasmcloud.yml` for other examples

Reply:
<.<